kinto_http/__init__.py

```diff
@@ -213,6 +215,9 @@ def create_bucket(self, bucket=None, data=None, permissions=None,
                                               safe=safe)
         headers = DO_NOT_OVERWRITE if safe else None
         endpoint = self.get_endpoint('bucket', bucket=bucket)
+
+        logger.info("Create bucket %r with data %r" % (bucket, data))
```

Comment:
I would probably log the data and permission only if it is present with another call to logger.
Maybe we can have an helper:

```
log_params(logger, "Create bucket", bucket=bucket, data=data, permissions=permissions, safe=safe, if_not_exists=if_not_exists)
```